arci/src/traits/navigation.rs

```diff
@@ -11,5 +11,18 @@ pub trait Navigation: Send + Sync {
         timeout: std::time::Duration,
     ) -> Result<WaitFuture, Error>;
 
+    #[allow(unused_variables)]
+    fn send_goal_pose_with_thresholds(
+        &self,
+        goal: Isometry2<f64>,
+        thresholds: [f64; 3],
```

Comment:
Add some comments about what the `thresholds` means, or 
change the name to goal_torrelance_x_y_theta?